src/Simulation/QsharpCore/Core.qs

```diff
@@ -30,6 +30,21 @@ namespace Microsoft.Quantum.Core {
     @Attribute()
     newtype Deprecated = (NewName : String);
 
+    /// # Summary
+    /// Compiler-recognized attribute used to mark a callable with the runtime capabilities it requires.
+    ///
+    /// # Input
+    /// ## Level
+    /// The name of the runtime capability level required by the callable.
```

Comment:
This API documentation section will not currently be formatted in documentation (see https://github.com/microsoft/qsharp-compiler/issues/113). If https://github.com/microsoft/qsharp-compiler/pull/594 is adopted, the format for named items in UDTs will be finalized as the following:
```suggestion
    /// # Named Items
    /// ## Level
    /// The name of the runtime capability level required by the callable.
```

Reply:
Very possibly, yeah. We don't yet have an official format for named items in UDTs due to https://github.com/microsoft/qsharp-compiler/issues/113, so it may be best to hold off updating until we have a format decided on.

Reply:
I went ahead and renamed `Input` to `Named Items` for both this attribute and `Deprecated`.